omod/src/main/webapp/resources/htmlforms/art_mastercard.xml

```diff
@@ -153,7 +153,11 @@
                         <tr>
                             <td>CHW Name</td>
                             <td>
-                                <obs conceptId="$chwName"  />
+                              <lookup complexExpression="#foreach(${relationship} in ${relationshipList})
+                                  #if(${relationship.relationshipType.bIsToA} == 'Patient')
```

Comment:
Hi @tlimbani , can we check here for relationshipType.uuid instead of bIsToA? I think that would be more safe on identifying the correct relationship type. Thanks!

Reply:
Hi @cioan thanks for the TIP and feedback! Yes this makes much more sense. I will add it in!

Reply:
I made the changes @cioan!